### PR TITLE
Improve the UI for repeated values.

### PR DESCRIPTION
**Pull request by nsthorat**
This is the chatml format:

image

Repeated:
https://lilacai-nikhil-staging.hf.space/datasets#local/OpenHermes-2.5
https://lilacai-nikhil-staging.hf.space/datasets#local/textbook_quality_programming
Non-repeated:
https://lilacai-nikhil-staging.hf.space/datasets#lilac/OpenOrca-100k

### CODE REVIEW

File: web/blueprint/src/lib/components/datasetView/RowItem.svelte [hunk 1]
```diff
@@ -124,6 +122,8 @@
       }
     });
   }
+
+  $: console.log('media fields:', mediaFields);
```
Comment (dsmilkov) [inline web/blueprint/src/lib/components/datasetView/RowItem.svelte]: remove console.log

File: web/blueprint/src/lib/components/datasetView/RowItem.svelte [hunk 2]
```diff
@@ -159,17 +159,16 @@
...
-            {@const path = L.path(valueNode) || []}
-            <div
-              class:border-b={i < mediaFields.length - 1}
-              class:pb-2={i < mediaFields.length - 1}
-              class="flex h-full w-full flex-col border-neutral-200"
-            >
-              <ItemMedia {row} {path} field={mediaField} {highlightedFields} />
-            </div>
-          {/each}
+          <!-- {#each valueNodes as valueNode} -->
```
Comment (dsmilkov) [inline web/blueprint/src/lib/components/datasetView/RowItem.svelte]: remove old code

**Review by dsmilkov — APPROVED** [review]

File: web/blueprint/src/lib/components/datasetView/RowItem.svelte (hunk 2 above)
Comment (nsthorat) [reply web/blueprint/src/lib/components/datasetView/RowItem.svelte]: done

File: web/blueprint/src/lib/components/datasetView/RowItem.svelte (hunk 1 above)
Comment (nsthorat) [reply web/blueprint/src/lib/components/datasetView/RowItem.svelte]: done

**Pull request merged**